### PR TITLE
Docs: Fix some typos

### PR DESCRIPTION
**Pull request by miles-six**
I have been studying this repository for a long time, and whenever I come across a typo, I fix it. 

I discovered 7 typos in a total of 7 files. Then I merged them together.

The code quality is exceptionally high, so sincerely hope my PR can help make this repository more standardized.

### CODE REVIEW

**Comment by miles-six** [comment]
I have singed CLA

File: docs/fundamentals/write_contract/README.md [hunk 1]
```diff
@@ -25,6 +25,6 @@ This will download and install the fhEVM Solidity Library and its dependencies i
 
 1/ For quick prototyping of a specific feature, use the [Zama version of the Remix IDE](remix.md). This will let you quickly deploy a contract on the devnet via Metamask, and interact easily with it through the Remix UI. Otherwise, for a bigger project, you should use our custom [`fhevm-hardhat-template` repository](https://github.com/zama-ai/fhevm-hardhat-template). Hardhat is a popular development environment for Solidity developers and will let you test and deploy your contracts to the fhEVM using TypeScript.
 
-2/ A good first step is to start with an unencrypted version of the contract you want to implement, as you would usually do on a regular EVM chain. It is easier to reason first on cleartext variables, before thinking on how to add confidentialy.
+2/ A good first step is to start with an unencrypted version of the contract you want to implement, as you would usually do on a regular EVM chain. It is easier to reason first on cleartext variables, before thinking on how to add confidentially.
```
Comment (jatZama) [inline docs/fundamentals/write_contract/README.md]: Please correct to : `confidentiality`

**Review by jatZama — CHANGES_REQUESTED** [review]

File: docs/fundamentals/write_contract/README.md (hunk 1 above)
Comment (miles-six) [reply docs/fundamentals/write_contract/README.md]: Thank you,already down it

Comment (jatZama) [reply docs/fundamentals/write_contract/README.md]: btw commit message must follow commitlint convention or else CI will fail.

Comment (miles-six) [reply docs/fundamentals/write_contract/README.md]: Can you tell me how to follow commitlint convention？Thank you sir

Comment (jatZama) [reply docs/fundamentals/write_contract/README.md]: typically you would start this commit with "docs: ..." see https://github.com/conventional-changelog/commitlint

Comment (miles-six) [reply docs/fundamentals/write_contract/README.md]: Thank you again sir, already down,Thank you

**Comment by immortal-tofu** [comment]
@cla-bot check

**Comment by miles-six** [comment]
@cla-bot check

File: docs/fundamentals/write_contract/README.md (hunk 1 above)
Comment (miles-six) [reply docs/fundamentals/write_contract/README.md]: Sir, sorry again,I have signed CLA for many times,but it still failed. I don't why

**Comment by miles-six** [comment]
@cla-bot check

**Comment by miles-six** [comment]
@cla-bot check

**Comment by miles-six** [comment]
45a53c198beb620c7b53862b07f10fdc
df0a071f3844d8e45ac48654801a9ec3
I have singed as the config, but I have failed again

**Comment by miles-six** [comment]
@cla-bot check

**Comment by aquint-zama** [comment]
Once the CLA is signed it's a manual process to update the CLA bot whitelist. I'll do it.

**Comment by miles-six** [comment]
Thank your response sir @aquint-zama 

Because it hasn't been approved for a while, I have signed multiple copies of the CLA.

 I hope this won't affect you.

**Comment by aquint-zama** [comment]
@cla-bot check

**Comment by aquint-zama** [comment]
> Thank your response sir @aquint-zama
> 
> Because it hasn't been approved for a while, I have signed multiple copies of the CLA.
> 
> I hope this won't affect you.

No don't worry, everything is ok now.

**Comment by aquint-zama** [comment]
updated in https://github.com/zama-ai/fhevm/pull/350

**Comment by aquint-zama** [comment]
merged in #350

**Pull request closed without merging**